brush/handler.go

```diff
@@ -26,7 +25,7 @@ type Handler struct {
 // NewHandler creates a new Handler for the *player.Player passed.
 func NewHandler(p *player.Player) *Handler {
 	h := &Handler{p: p}
```

Comment:
I would also suggest removing the `*player.Player` from the handler. This is passed to the handler methods now.